src/EnergyPlus/CMakeLists.txt

```diff
@@ -570,11 +570,16 @@ install( TARGETS energyplusapi
   LIBRARY DESTINATION ./ 
 )
 
-if (CMAKE_COMPILER_IS_GNUCXX AND ENABLE_LTO)
+if (CMAKE_COMPILER_IS_GNUCXX)
```

Comment:
@nealkruis @lefticus @Myoldmopar I have fiddled with the logic here to get MinGW to hit the target_link_libraries lines that follow. Without that, I get linker errors on the energyplusapi target. I don't think this is the correct solution, I am not sure why it is needed in the first place, and I am reluctant to merge something that doesn't make sense to me. Any ideas would be appreciated.